### PR TITLE
28 forward headers by the triggerchallenge

### PR DESCRIPTION
**Pull request by lukasmatusiewicz**
- fixed forwarding headers by the triggerchallenge
- updated installation process to automatically create the config file template
- updated readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,33 +7,40 @@
...
+2. **Open the terminal and run: `sudo -i` for the admin rights.**
 3. **Enable the MFA Module: `$idp_install_path/bin/module.sh -t idp.authn.MFA || $idp_install_path/bin/module.sh -e idp.authn.MFA`.**
 4. **Check if the MFA Module is activated: `$idp_install_path/bin/module.sh -l`.**
 5. **Install the privacyIDEA Plugin: `$idp_install_path/bin/plugin.sh -i path/to/zip/from/step/1 --noCheck`**<br>
-The installer will also install and enable the privacyIDEA Module. You can check it by repeating the step 4.
-6. **Check if the privacyIDEA Plugin is installed correctly: `$idp_install_path/bin/plugin.sh -l`**<br>
-If you see the *org.privacyidea.privacyIDEA* on the list, the plugin is installed successfully.<br>
-Otherwise, re-run the command from step 5.
+Note: The installer will automatically install and enable the privacyIDEA Module. You can check it state by repeating the step 4.
+6. **Check if the privacyIDEA Plugin is installed propertly: `$idp_install_path/bin/plugin.sh -l`**<br>
```
Comment (nilsbehlen) [inline README.md:16]: properly or correctly

File: README.md [hunk 2]
```diff
@@ -7,33 +7,40 @@
...
-The different configuration parameters that are available on the configuration page of the execution are explained in the following table:
+2. **Add the privacyIDEA subflow to the MFA flow.**<br>
+   - Path to the MFA flow configuration file: `$idp_install_path/conf/authn/mfa-authn-config.xml`.
+   - Example of the *util:map* is located in the *privacyidea.properties* file (`$idp_install_path/conf/authn/privacyidea.properties`).
+   - Remember to activate the MFA flow.
+
+3. **Turn on the MFA Module by updating following file: `$idp_install_path/conf/authn/authn.properties`.**<br>
+Note: Example of this configuration contains the *privacyidea.properties* file (`$idp_install_path/config/authn/privacyidea.properties`).
+
+The different configuration parameters, are explained in the following table:
```
Comment (nilsbehlen) [inline README.md:43]: remove ','

File: README.md [hunk 3]
```diff
@@ -7,33 +7,40 @@
...
-   - `privacyidea.verify_ssl=true`
-   - `privacyidea.default_message=Please enter the OTP`
-   - `privacyidea.otp_field_hint=OTP`
-   - `privacyidea.triggerchallenge=false`
-   - `privacyidea.service_name=service`
-   - `privacyidea.service_pass=service`
-   - `privacyidea.service_realm=defrealm`
-   - `privacyidea.forward_headers=header1,header2,header3`
-   - `privacyidea.debug=false`
+1. **Update the *privacyidea.properties* file (`$idp_install_path/config/authn/privacyidea.properties`), by adding your own configuration data.**<br>
```
Comment (nilsbehlen) [inline README.md:21]: remove ','

**Review by nilsbehlen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lukasmatusiewicz) [reply README.md]: fixed: [b8d27f4](https://github.com/privacyidea/shibboleth-plugin/pull/32/commits/b8d27f4079342b8385b153bc378226c17a652136)

File: README.md (hunk 2 above)
Comment (lukasmatusiewicz) [reply README.md]: fixed: [b8d27f4](https://github.com/privacyidea/shibboleth-plugin/pull/32/commits/b8d27f4079342b8385b153bc378226c17a652136)

File: README.md (hunk 3 above)
Comment (lukasmatusiewicz) [reply README.md]: fixed: [6596f16](https://github.com/privacyidea/shibboleth-plugin/pull/32/commits/6596f1613ee8555930938857d881645e71dd4798)

**Review by nilsbehlen — APPROVED** [review]

**Pull request merged**